docs/_includes/header.html

```diff
@@ -15,14 +14,11 @@
       </li>{% endif %}  -->
 
         <!-- Internal links -->
-        {% for link in site.data.navigation %} 
-            <li class="nav-item mr-4 mb-2 mb-lg-0">
-              <a
-                class="nav-link"
-                href="{% if link.url %}{{ site.baseurl }}/{{ link.url }}{% else %}{{ link.external_url }}{% endif %}"
-                ><span>{{ link.title }}</span></a
-              >
-            </li>
+        {% for link in site.data.navigation %}
+        <li class="nav-item mr-4 mb-2 mb-lg-0">
+          <a class="nav-link"
+            href="{% if link.url %}{{ site.baseurl }}/{{ link.url }}{% else %}{{ link.external_url }}{% endif %}"><span>{{ link.title }}</span></a>
+        </li>
```

Comment:
@msaini0r Could you please revert this change as it just formats this file, or it would be great if you could run a lint fix as well :)

CC @warunicorn19